examples/docs_snippets/docs_snippets/guides/dagster/asset_tutorial/cereal.py

```diff
@@ -12,7 +12,7 @@
 
 
 @asset
-def cereals():
+def cereals():  # type: ignore (didactic)
```

Comment:
Why is this a type error?

Reply:
Because the same function is redefined in the same scope below.